integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -17,7 +18,79 @@ All examples in this starter kit require that you use the [new IBM watsonx Assis
 
 Create a new, empty assistant that you can use to test this starter kit. For more information, see [Adding more assistants](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-assistant-add).
 
-## Example 1: Connect your assistant to Watson Discovery and watsonx via custom extensions
+## Example 1: Connect your assistant to Elasticsearch and watsonx via custom extensions 
+
+This starter kit example shows how to configure your assistant to use Elasticsearch for document search, and then use 
+those search results as input context for a watsonx large language model (LLM). The watsonx LLM generates a natural 
+language answer for the query based on the documents provided by the search.  
+
+Before you start, you need to install and set up your Elasticsearch index. 
+Please refer to [elasticsearch-install-and-setup-guide](../../docs/elasticsearch-install-and-setup/ICD_Elasticsearch_install_and_setup.md) for more details. 
+
+Then follow the steps in the following two sections to configure your custom extensions. 
+
+### Configure Elasticsearch extension 
+Follow the steps [here](../elasticsearch/README.md) to configure the Elasticsearch custom extension
+
+### Configure the watsonx answer generation extension 
+Follow the steps [here](../language-model-watsonx/README.md) to configure watsonx as a custom extension.
+
+### Upload sample actions
+The starter kit includes [a JSON file with these sample actions](./elasticsearch-watsonx-actions.json):  
+
+
+| Action                        | Description                                                                                                                                                                                                                                                                                                  |
+| ----------------------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Search                        | Connects to Elasticsearch to search for documents related to the user query. The "No Action Matches" action has been configured to send all input to this action, so whatever the user enters will be used as the search input. It invokes the "Generate Answer" action to generate a response to the query. |
+| Generate Answer               | Configures the query prompt and document passages resulting from search, and calls the action "Invoke watsonx generation API". It is not meant to be invoked directly, but rather by the "Search" action.                                                                                                    |
```

Comment:
The Generate Answer action uses the _full_ text of the search results and sends it along to the models.  This works well if the documents in your index are quite short.  That can be true if your original source documents are very short _or_ you have split your original source documents into smaller chunks and have indexed those chunks as separate documents in Elastic.  If not, you might need to provide a query body that asks for highlights from Elasticsearch and update the logic of Generate Answer to use the highlights and not the full text.  I don't think we want to actually _do_ that in this kit, but we should mention it in the README.

Reply:
I added some more description about the sample data we used in this kit [here](https://github.com/watson-developer-cloud/assistant-toolkit/pull/233/files#diff-40d59f42b193719a41b6d5644f417ea9362b5922595cb4144268b6610297e39bR132-R137).

I am not sure if mentioning the Elasticsearch `Highlighting` feature in this starter kit is a good idea. It could be confusing to the user if there are too many paths/options in the starter kit, IMO.